CMakeLists.txt

```diff
@@ -26,7 +26,7 @@
 
 # build engine library and all tests
 
-cmake_minimum_required(VERSION 3.6)
+cmake_minimum_required(VERSION 3.14)
```

Comment:
So we should use documentation to let developers know cmake 3.14+ is needed for iOS?

Reply:
There are to options:
1. Use cmake 3.14 everywhere
That means update Travis to cmake 3.14+ and Android config
```
android {
   externalNativeBuild {
       cmake {
           version "3.14+"
       }
   }
```
2. Update cmake only for iOS 
And add apple/ios specific error
```
If(APPLE)
Iif(${CMAKE_VERSION} VERSION_LESS "3.14") 
    message(FATAL_ERROR "Please use  CMake 3.14 or newer")
endif()
endif()
```

I do not have here strong opinion what is better but... Because Android development is hard and for now cmake 3.6 is default used by Android studio I think that it is safer to use cmake from Android SDK (option 2)

Reply:
Sounds reasonable.